controller/hydrator/types/types.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+type HydrationQueueKey struct {
```

Comment:
I had to move this to its own package to avoid circular dependencies between the hydrator package and the mocks package.